eip-0019.md

```diff
@@ -175,16 +175,33 @@ For better security and usability additional data can be transfered between Hot
 and Cold wallets via QR codes. Hot Wallet passes data to the Cold Wallet using
 ColdSigningRequest data format and Cold Wallet replies back with ColdSigningResponse.
 
-```
-ColdSigningRequest:
- - messageType: Byte = 1
- - transaction: ReducedTransaction
- - description: String
-
-ColdSigningResponse:
- - messageType: Byte = 2
- - transaction: SignedTransaction
-```
+### ColdSigningRequest
+Json format, holding
+* reducedTx (mandatory): Base64-encoded `ReducedTransaction` as defined above
+* sender (optional): P2PK address sending the transaction, can be used by cold wallets to determine which secret to use for signing the transaction
+* inputs (optional): List of base64-encoded serialized input boxes, can be used by cold wallets to show the user which boxes are burnt
+
+### ColdSigningResponse:
+Json format, holding
+* signedTx (mandatory): Base64-encoded `SignedTransaction`
+
+### Interchange format
+As QR codes are length-limited and it could be needed to transfer the data in chunks, it is needed to wrap the actual data sent by a small layer containing information about number of chunk pages and page index of a chunk. This is done by adding a prefix to the data defined above:
+
+* Constant CSR for ColdSigningRequest or CSTX for ColdSigningResponse
+* (if pages num > 1) const / and 1-based page index of current chunk
+* (if pages num > 1) const / and pages num
+* const -
+
+Examples:
+The QR code for a complete ColdSigningRequest contains the following data:
+
+     CSR-{"reducedTx":"....","sender":"9...","inputs":["...."]}
```

Comment:
I agree, it is easier to implement that way. I aimed to keep the payload as small as possible, but when using just p and n this is small as well.

Reply:
It's now wrapped into JSON instead of an own prefix. Please note that the inner JSON has to be a String and not a JSON object due to the fact that for chunks it isn't a valid JSON object.